qiskit_dynamics/type_utils.py

```diff
@@ -320,39 +322,125 @@ def vec_dissipator(L: Array):
     )
 
 
-def to_array(op: Union[Operator, Array, List[Operator], List[Array], spmatrix]):
-    """Convert an operator or list of operators to an Array.
+def isinstance_qutip_qobj(obj):
+    """Check if the object is a qutip Qobj.
+
+    Bool: True if obj is qutip Qobj
+    """
+    if (
+        type(obj).__name__ == "Qobj"
+        and hasattr(obj, "_data")
+        and type(obj._data).__name__ == "fast_csr_matrix"
+    ):
+        return True
+    return False
 
+
+def to_array(op: Union[Operator, Array, List[Operator], List[Array], spmatrix], no_iter=False):
+    """Convert an operator or list of operators to an Array.
     Args:
         op: Either an Operator to be converted to an array, a list of Operators
             to be converted to a 3d array, or an array (which simply gets
             returned)
+        no_iter (Bool): Boolean determining whether to look inside iterables to make multiple
```

Comment:
I think this explanation of `no_iter` is a bit unclear. You could say something as simple as: "Whether to recursively unroll `Iterables`."